### PR TITLE
Marketplace add docs links

### PR DESCRIPTION
**Pull request by reutshal**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/36826

## Description
* Update the packs details generation to run in parallel.
* Created a file containing all the existing README file, try to find the readme there, and id it does not exist - perform an HTTP requests to the URL in will the README should be.
@sserrata FYI

### CODE REVIEW

File: genMarketplace.js [hunk 1]
```diff
@@ -46,6 +50,61 @@ function capitalizeFirstLetter(string) {
...
+    if (response.ok) {
+      listItem.docLink = url;
+    }
+    else {
+      listItem.docLink = ""
+    }
+  }).catch(err => listItem.docLink = "");
+}
+
+function normalizeItemName(itemName) {
```
Comment (Itay4) [inline genMarketplace.js:65]: note this does not work properly
for example, the link from 
image
leads to https://xsoar.pan.dev/docs/reference/integrations/mitre-attck-feed which does not exist
in general im not fan of having this normalization functionality in both python and javascript

Comment (reutshal) [reply genMarketplace.js:65]: @Itay4 I think you were looking at the old preview site, in the new one there aren't supposed to be broken links 
image
 But I do see there is a README for this which we don't link to.
I tried using the same library the python's logic but it resulted differently.
I think I will create a file containing all the links (created be the python logic) and will maintain it.

File: genMarketplace.js [hunk 2]
```diff
@@ -46,6 +50,61 @@ function capitalizeFirstLetter(string) {
   }
 }
 
+function checkURLAndModifyLink(url, listItem) {
```
Comment (yaakovi) [inline genMarketplace.js:53]: don't forget to remove this part when the content-gold PR is done

**Review by yaakovi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -74,8 +74,12 @@ This task will download the packs information from the google cloud bucket and g
 Additional env vars that affect the generation of the docs:
 * `MAX_PACKS`: max packs to generate in the marketplace tab. Set this for faster generation when developing. Note also in CI builds on branches this is automatically set to speed up the preview site if there are only modifications related to docs. For example:
   
-  export MAX_PACKS=10 && npm run marketplace-docs
-  ```
+  ```export MAX_PACKS=10 && npm run marketplace-docs```
+  
+```
+NOTE : When running locally, in order to see links from the entities in the marketplace to their correspond README in the reference section, you will need to run the reference-docs script before running the marketplace-docs script.
```
Comment (yaakovi) [inline README.md:80]:
```suggestion
NOTE : When running locally, in order to see links from the entities in the marketplace to their corresponding README in the reference section, you will need to run the reference-docs script before running the marketplace-docs script.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**